Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.7-buster
```

Comment:
What about just "python:3" ? (once we support 3.8)

Reply:
Ah, we fixed 3.8 ?
(not sure we published since)

Reply:
3.8 is not fixed on pypi (yet), but as this image is used within docker-compose where we install the repository version, it works. :)